torch/distributions/bernoulli.py

```diff
@@ -25,27 +27,29 @@ class Bernoulli(Distribution):
 
     def __init__(self, probs):
         self.probs, = broadcast_all(probs)
+        if isinstance(probs, Number):
+            batch_shape = ()
```

Comment:
Will this conditional be unnecessary once scalars are supported?

Reply:
That shouldn't be necessary, because broadcast_all would then return a Variable of dim 0.